_content/monographs/walking-meditation-story_karunaratna-suvimalee.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "The Walking Meditation: A Story"
+authors:
+  - "Suvimalee Karunaratna"
+external_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1BJVhXDAZA5yVSrZhA3ajtHO_uOfR3aIW/view?usp=sharing"
```

Comment:
Add the epub as a second link: https://drive.google.com/file/d/1A-TQThyZjc2062h_4u89ACsNnRq4bwbd/view?usp=drivesdk